### ISSUE TITLE
Discord bridge silently fails to upload file larger than 8MB to Discord server. 

### ISSUE THREAD

**Issue by DarkLinkXXXX** [Half-Shot/matrix-appservice-discord#109]
In my opinion, the bot should notify the sender through a PM of the issue, and have an option to notify the entire room.

**Comment by Half-Shot** [comment]
Ooor just link the content url as a message, which I believe the IRC bridge does.

**Comment by Mikaela** [comment]
How will you handle redactions? If I recall corectly, the IRC bridge doesn't and there are no warnings of the link working forever.

**Comment by Half-Shot** [comment]
We remove messages the usual way. If you are talking about removing media, we can't. I don't know if there is a general purpose media removal AP yet for users (not admins) but I can't really do much about it. Media once uploaded sticks forever without manual intervention.

**Comment by TildeSlashC0re** [comment]
I am experiencing the same so here's some log-output for sending a 13mb sized gif:

```node
ERR! DiscordBot DiscordAPIError: Request entity too large
ERR! DiscordBot     at DiscordAPIError (/home/c0re/matrix-appservice-discord/node_modules/discord.js/src/client/rest/DiscordAPIError.js:7:5)
ERR! DiscordBot     at item.request.gen.end (/home/c0re/matrix-appservice-discord/node_modules/discord.js/src/client/rest/RequestHandlers/Sequential.js:71:65)
ERR! DiscordBot     at then (/home/c0re/matrix-appservice-discord/node_modules/snekfetch/src/index.js:215:21)
ERR! DiscordBot     at process._tickCallback (internal/process/next_tick.js:103:7)
ERR! DiscordBot  Couldn't send message.  { DiscordAPIError: Request entity too large
ERR! DiscordBot     at DiscordAPIError (/home/c0re/matrix-appservice-discord/node_modules/discord.js/src/client/rest/DiscordAPIError.js:7:5)
ERR! DiscordBot     at item.request.gen.end (/home/c0re/matrix-appservice-discord/node_modules/discord.js/src/client/rest/RequestHandlers/Sequential.js:71:65)
ERR! DiscordBot     at then (/home/c0re/matrix-appservice-discord/node_modules/snekfetch/src/index.js:215:21)
ERR! DiscordBot     at process._tickCallback (internal/process/next_tick.js:103:7)
ERR! DiscordBot   stack: 'DiscordAPIError: Request entity too large\n    at DiscordAPIError (/home/c0re/matrix-appservice-discord/node_modules/discord.js/src/client/rest/DiscordAPIError.js:7:5)\n    at item.request.gen.end (/home/c0re/matrix-appservice-discord/node_modules/discord.js/src/client/rest/RequestHandlers/Sequential.js:71:65)\n    at then (/home/c0re/matrix-appservice-discord/node_modules/snekfetch/src/index.js:215:21)\n    at process._tickCallback (internal/process/next_tick.js:103:7)',
ERR! DiscordBot   name: 'DiscordAPIError',
ERR! DiscordBot   message: 'Request entity too large',
ERR! DiscordBot   path: '/api/v7/channels/429209802493919243/messages',
ERR! DiscordBot   code: 40005 }
verb DiscordBot Sent  null
WARN MatrixRoomHandler TypeError: Cannot read property 'id' of null
WARN MatrixRoomHandler     at msg.forEach (/home/c0re/matrix-appservice-discord/build/src/bot.js:247:41)
WARN MatrixRoomHandler     at Array.forEach (native)
WARN MatrixRoomHandler     at DiscordBot.<anonymous> (/home/c0re/matrix-appservice-discord/build/src/bot.js:245:17)
WARN MatrixRoomHandler     at throw (native)
WARN MatrixRoomHandler     at rejected (/home/c0re/matrix-appservice-discord/build/src/bot.js:5:65)
WARN MatrixRoomHandler     at process._tickCallback (internal/process/next_tick.js:103:7)
WARN MatrixRoomHandler  There was an error sending a matrix event { TypeError: Cannot read property 'id' of null
WARN MatrixRoomHandler     at msg.forEach (/home/c0re/matrix-appservice-discord/build/src/bot.js:247:41)
WARN MatrixRoomHandler     at Array.forEach (native)
WARN MatrixRoomHandler     at DiscordBot.<anonymous> (/home/c0re/matrix-appservice-discord/build/src/bot.js:245:17)
WARN MatrixRoomHandler     at throw (native)
WARN MatrixRoomHandler     at rejected (/home/c0re/matrix-appservice-discord/build/src/bot.js:5:65)
WARN MatrixRoomHandler     at process._tickCallback (internal/process/next_tick.js:103:7)
WARN MatrixRoomHandler   stack: 'TypeError: Cannot read property \'id\' of null\n    at msg.forEach (/home/c0re/matrix-appservice-discord/build/src/bot.js:247:41)\n    at Array.forEach (native)\n    at DiscordBot.<anonymous> (/home/c0re/matrix-appservice-discord/build/src/bot.js:245:17)\n    at throw (native)\n    at rejected (/home/c0re/matrix-appservice-discord/build/src/bot.js:5:65)\n    at process._tickCallback (internal/process/next_tick.js:103:7)' }
```

**Comment by Half-Shot** [comment]
Thanks @TildeSlashC0re

**Comment by Half-Shot** [comment]
Fixed by #116
